tests/core/test_load.py

```diff
@@ -69,3 +150,50 @@ def test_no_loader(zip_file):
     artifact = ln.Artifact(zip_file, key=str(zip_file))
     with pytest.raises(NotImplementedError):
         artifact.load()
+
+
+def test_load_csv(csv_filepath):
+    artifact = ln.Artifact(csv_filepath, key=str(csv_filepath))
+    df = artifact.load()
+    assert df.iloc[0, 0] == 1
+    assert df.iloc[0, 1] == 2
+
+
+def test_load_tsv(tsv_filepath):
+    artifact = ln.Artifact(tsv_filepath, key=str(tsv_filepath))
+    df = artifact.load()
+    assert df.iloc[0, 0] == 1
+    assert df.iloc[0, 1] == 2
+
+
+def test_load_parquet(parquet_filepath):
+    artifact = ln.Artifact(parquet_filepath, key=str(parquet_filepath))
+    df = artifact.load()
+    assert df.iloc[0, 0] == 1
+    assert df.iloc[1, 1] == 4
+
+
+def test_load_yaml(yaml_filepath):
+    artifact = ln.Artifact(yaml_filepath, key=str(yaml_filepath))
+    data = artifact.load()
+    assert data["a"] == 1
+    assert data["b"] == 2
+
+
+def test_load_image(image_filepath):
+    artifact = ln.Artifact(image_filepath, key=str(image_filepath))
+    result = artifact.load()
+    assert Path(result).name == image_filepath.name
+
+
+def test_load_svg(svg_filepath):
+    artifact = ln.Artifact(svg_filepath, key=str(svg_filepath))
+    result = artifact.load()
```

Comment:
Why doesn't this error? Because the loader is configured to display an image and that will only work in ipython?

Reply:
Correct. Else, it just returns a Path which I think is somewhat weird or inconsistent but not the scope if this PR.

```python
def load_svg(path: UPathStr) -> None | UPathStr:
    """Display `.svg` in ipython, otherwise return path."""
    if is_run_from_ipython:
        from IPython.display import SVG, display

        display(SVG(filename=path))
        return None
    else:
        return path
```